.github/workflows/ci.yaml

```diff
@@ -19,16 +19,13 @@ jobs:
     strategy:
       matrix:
         base-image:
-          - nvidia/cuda:11.8.0-cudnn8-devel-ubuntu22.04
-          - nvidia/cuda:11.8.0-cudnn8-runtime-ubuntu22.04
+          - nvidia/cuda:12.6.2-cudnn-devel-ubuntu24.04
+          - nvidia/cuda:12.6.2-cudnn-devel-ubuntu24.04
```

Comment:
Previously, we were considering keeping two versions of base images -- runtime and devel.

The reason was mainly because the image size (devel is ~ 4 gigs larger, if I'm not mistaken).

The devel is rarely needed imo, users pick it only if there's a need to build some frameworks from sources. Otherwise, runtime is good to go.

Maybe, we could keep runtime version too, wdyt?